duo-universal-sdk/src/main/java/com/duosecurity/service/DuoConnector.java

```diff
@@ -22,24 +25,35 @@ public class DuoConnector {
   /**
    * DuoConnector Constructor.
    *
-   * @param apiHost         This value is the api host provided by Duo in the admin panel.
-   * @param caCerts         CA Certificates used to connect to Duo
+   * @param apiHost This value is the api host provided by Duo in the admin panel.
+   * @param caCerts CA Certificates used to connect to Duo
    *
-   * @throws DuoException   For issues getting and validating the URL
+   * @throws DuoException For issues getting and validating the URL
    */
   public DuoConnector(String apiHost, String[] caCerts) throws DuoException {
-    CertificatePinner duoCertificatePinner = new CertificatePinner.Builder()
-        .add(apiHost, caCerts)
-        .build();
+    this(apiHost, null, null, caCerts);
+  }
 
-    OkHttpClient client = new OkHttpClient.Builder().certificatePinner(
-                            duoCertificatePinner).build();
```

Comment:
I think the cert pinning part of this `DuoConnector` is deleted by accident? I believe we still want cert pinning function even without have proxy function

Reply:
Hi @yizshi. Thanks for reviewing the PR. The CertificatePinner is created on line 48. So it was moved, but not deleted entirely. If a user doesn't want to specify a proxy, the original constructor remains and calls the new DuoConnector constructor with null values for proxyHost and proxyPort. I'm sure there are other ways to accomplish the same result. I believe this approach will ensure that existing code will not break.